JS_6/task2.3.js

```diff
@@ -0,0 +1,22 @@
+// 3. Написать функцию, которая будет добавлять предприятие. В качестве аргумента принимает название предприятия
+
+// Пример:
+// addEnterprise("Название нового предприятия")
+
+import { enterprises } from "./task2.1.js";
+
+const addEnterprise = (name => {
+
+    const newId = Math.max(...enterprises.map(enterprise => enterprise.id)) + 1;
```

Comment:
Incorrect. New id must be found through all departments too. All ids must be unique among all objects.

And of cause this code should be moved to an helper function generateNewId

Reply:
understood, I've fixed it